arch/arm/core/fault.c

```diff
@@ -268,8 +290,13 @@ static u32_t _MpuFault(const NANO_ESF *esf, int fromHardFault)
  *
  * @return N/A
  */
-static void _BusFault(const NANO_ESF *esf, int fromHardFault)
+static int _BusFault(NANO_ESF *esf, int fromHardFault)
 {
+	/* Assess whether system shall ignore/recover from this MPU fault. */
+	if (_MemoryFaultIsRecoverable(esf)) {
+		return _NANO_ERR_RECOVERABLE;
```

Comment:
The immediate `return `here is a bit problematic, because we need to clear the sticky bits, see line 386-371. I see two "good " options:
1. Copy here the code for sticky bits clearing (note that it is only relevant for ARMv8-M), or
2.  Create a local variable `reason `to hold the `_NANO_ERR_RECOVERABLE`, and jump with `goto `to the  sticky bit clearing and then `return reason`.